AutoMapper.Data/DataReaderProfileBase.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection;
+using AutoMapper.Configuration.Conventions;
+using AutoMapper.Data.Configuration.Conventions;
+using AutoMapper.Mappers;
+
+namespace AutoMapper.Data
+{
+    public abstract class DataReaderProfileBase : Profile
```

Comment:
I think maybe this is a step too far :) I appreciate the intent, but too many ways of doing the same thing can be confusing, convenience aside. Instead of changing the base class, you cab simply call the extension method. Seems simple enough.

Reply:
The abstract class resolves the issue reported (#35). It is intended for use with dependency injection and assembly scanning. I added the convenience method to facilitate one-off configuration.